backend/pkg/logger/logger.go

```diff
@@ -28,24 +29,44 @@ type Logger struct {
 
 var _ abstraction.Logger = &Logger{}
 
-var Timestamp time.Time = time.Now()
+var Timestamp = time.Now()
 
 func (Logger) HandlerName() string {
 	return HandlerName
 }
 
-func (logger Logger) UpdateMessage(client wsModels.Client, message wsModels.Message) {
+func (logger Logger) UpdateMessage(_ wsModels.Client, message wsModels.Message) {
 	var enable bool
 	err := json.Unmarshal(message.Payload, &enable)
 	if err != nil {
-		logger.Stop()
+		err := logger.Stop()
+		if err != nil {
+			fmt.Printf(ErrStoppingLogger{
+				Name:      Name,
+				Timestamp: time.Now(),
+			}.Error())
+		}
 		fmt.Printf("Error unmarshalling enable message")
```

Comment:
forgot to see this the last time, but an invalid message shouldnt stop the logger, instead just print the unmarshaling error and return